medicines/doc-index-updater/src/create_manager/sftp_client.rs

```diff
@@ -20,7 +20,9 @@ pub enum SftpError {
 async fn sentinel_sftp_factory() -> Result<Sftp, SftpError> {
     let server = get_env_fail_fast("SENTINEL_SFTP_SERVER").await;
     let user = get_env_fail_fast("SENTINEL_SFTP_USERNAME").await;
-    let password = get_env_fail_fast("SENTINEL_SFTP_PASSWORD").await;
+    let public_key_path = get_env_fail_fast("SENTINEL_PUB_KEY_PATH").await;
```

Comment:
nitpick: PUB -> PUBLIC since using PRIVATE for the others ?

Reply:
:+1: and `public_key_path` below.